fvm/programs/block_programs.go

```diff
@@ -20,6 +26,10 @@ type DerivedTransactionData struct {
 		common.AddressLocation,
 		*interpreter.Program,
 	]
+
+	// There's only a single entry in this table.  For simplicity, we'll use
+	// 0 as the entry's key.
+	meterParamOverrides *TransactionDerivedData[int, MeterParamOverrides]
```

Comment:
maybe `struct{}{}`?

Reply:
o, i didn't know that works as a key.  I'll switch to that.